### DISCUSSION TITLE
Header

### DISCUSSION THREAD

**Discussion by MrJelly17** [matt8707/ha-fusion#396] [General]
Hiya, I have to say that this dashboard is the best dashboard ever. I have been using this for a few weeks now and it is performing brilliant. There is still a lot to do but you are on the right track. is there a way to hide the header?  It's just just doesn't match the colour came of the rest of the dashboard.

**Comment by Breakingx** [comment]
Hey,

could you provide a screenshot of what you would like to hide?

**Comment by MrJelly17** [comment]
![image](https://github.com/matt8707/ha-fusion/assets/68510016/9d1cde60-54c7-4489-9ae4-b4e2c300c63b)

**Comment by MrJelly17** [comment]
I just realised that if you access the fusion dashboard through Home Assistant, the header is visible, but if you access fusion dashboard after setting up external port  the dashboard is visible without Heather

**Reply by joakimtrulsson** [reply]
Hello,

I've encountered an issue with Fusion. When I access the dashboard through homeassistant.local:5050, all devices and entities appear as Unknown. However, when I access the Fusion dashboard through Home Assistant everything works as expected, and the devices/entities are displayed correctly.

Is there something specific I need to configure for external access, or could this be a permissions issue?
